ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+> ℹ Please fill out this template when filing an issue.
+> All lines beginning with a quotation mark (`>`),
+> instruct you with what info we expect.  
+> Please remove this opening paragraph.
+
+### Goals
+
+> ℹ Please replace this with what you tried to achieve.
+
+### Expected Results
+
+> ℹ Please replace this with what you expected to happen.
+
+### Actual Results
+
+> ℹ Please replace this with what actually happened.  
+> e.g. the stack trace of a crash
```

Comment:
In the interests of readability for us, could we please encourage people to put things like _stack traces_ at the **end** of the issue? Maybe have a final section called **Supporting Details** and have the advice here to be something like _Please provide a short description here and append lengthy information such as stack traces at the end of the issue_